packages/node_modules/@webex/widget-recents/src/enhancers/setup.js

```diff
@@ -128,12 +130,14 @@ function getInitialSpaces(props) {
     widgetStatus
   } = props;
 
+  const spaceLoadCount = props.spaceLoadCount || DEFAULT_SPACE_COUNT;
```

Comment:
```suggestion
  const spaceLoadCount = (props.spaceLoadCount > 0) ? props.spaceLoadCount : DEFAULT_SPACE_COUNT;
```
if the user passes in `'not the correct value'` or any other string

Reply:
At what point does this become over-engineering? lol